src/bindings/swig/python/CMakeLists.txt

```diff
@@ -45,18 +45,19 @@ else ()
 	# specify the SWIG_TYPE_TABLE to use (has to be in sync with the python plugin)
 	set (SWIG_COMPILE_FLAGS "${SWIG_COMPILE_FLAGS} -DSWIG_TYPE_TABLE=kdb")
 
+	# TODO: Reenable the following warning after we add a virtual destructor to `PluginDatabase`, and its subclasses.
+	# See also:
+	#
+	# * https://github.com/ElektraInitiative/libelektra/pull/1841
```

Comment:
Actually the "-" bullet points were correct according to the cmake format file, but let's see what it says now.

Reply:
hahaha